hashes/src/lib.rs

```diff
@@ -253,35 +253,3 @@ impl fmt::Display for FromSliceError {
 
 #[cfg(feature = "std")]
 impl std::error::Error for FromSliceError {}
```

Comment:
In 8b5958a6bcc18e71d9fe3db3979b4c87b3e8733a:

re "is anybody outside of rust-bitcoin using these macros" yes, we are using it in rust-miniscript for one.

Reply:
and elements-miniscript (in different ways than in rust-miniscript)